skills/lookup.js

```diff
@@ -0,0 +1,53 @@
+const ExpressionList = require('../submodules/Regex/ExpressionList');
+const ping = require('ping');
+
+
+module.exports = controller => {
+    controller.hears(
+        [ExpressionList.lookup], 
+        'direct_message', 
+        async (bot, message) => {
+            bot.createConversation(message, async(err, convo) => {
+                if (err) return false;
+                var domain = message.text.split(' ')[1];
+
+                if(domain == undefined){
+                    responseMessage = ("Missing parameter. Please provide the domain name to lookup");
+                }else{
+                    //Slack will automatically try to make anything ending in '.com' be a link
+                    //This causes issue with message response formatting which needs to be corrected
+                    if(domain[0] === '<'){ 
+                        // RegEx matches: <http:// and >
+                        var regex = /(<http:\/\/|>)/gi;
+                        domain = domain.replace(regex, '');
+                        domain = domain.split('|')[0];
+                    }
+                    let ip = await pingResult(domain);
+                    console.log("pingResult: " + ip);
+                    if(ip == undefined){
+                        responseMessage = "Could not retreive IP of domain: " + 
```

Comment:
Just saw it seems `responseMessage` is never created before you assign a value to it. In JS this can create globals, so I recommend somewhere inside the bot.createConversation callback, just make the var:
`let responseMessage;`